src/SDKs/CognitiveServices/management/Management.CognitiveServices/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Cognitive Services Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Cognitive Services management functions for managing Microsoft Azure Cognitive Services accounts.")]
 
-[assembly: AssemblyVersion("2.0.0.0")]
-[assembly: AssemblyFileVersion("2.0.0.0")]
+[assembly: AssemblyVersion("2.1.0.0")]
```

Comment:
@felixwa please do not update AssemblyVersion when you change the minor version of AssemblyFileVersion.

Reply:
@shahabhijeet , please note there is a breaking change here (provisionState is changed to string from enum), why do not increase AssemblyVersion in this case?